L1Trigger/TrackFindingTMTT/src/StubWindowSuggest.cc

```diff
@@ -0,0 +1,138 @@
+#include "L1Trigger/TrackFindingTMTT/interface/StubWindowSuggest.h"
+#include "L1Trigger/TrackFindingTMTT/interface/Stub.h"
+#include "L1Trigger/TrackFindingTMTT/interface/TrackerModule.h"
+#include "L1Trigger/TrackFindingTMTT/interface/PrintL1trk.h"
+
+#include "DataFormats/DetId/interface/DetId.h"
+#include "FWCore/Utilities/interface/Exception.h"
+
+#include <sstream>
+#include <mutex>
+
+using namespace std;
+
+namespace {
+  std::mutex myMutex;
```

Comment:
move to be a `static` data member inside `StubWindowSuggest::process` ?

Reply:
done